pkg/actions/nodegroup/get.go

```diff
@@ -58,27 +100,183 @@ func (m *Manager) GetAll() ([]*manager.NodeGroupSummary, error) {
 	return summaries, nil
 }
 
-func (m *Manager) Get(name string) (*manager.NodeGroupSummary, error) {
-	summaries, err := m.stackManager.GetUnmanagedNodeGroupSummaries(name)
+func (m *Manager) getUnmanagedSummaries() ([]*Summary, error) {
+	stacks, err := m.stackManager.DescribeNodeGroupStacks()
 	if err != nil {
-		return nil, errors.Wrap(err, "getting nodegroup stack summaries")
+		return nil, errors.Wrap(err, "getting nodegroup stacks")
 	}
 
-	if len(summaries) > 0 {
-		s := summaries[0]
-		if s.DesiredCapacity > 0 {
-			s.Version, err = kubewrapper.GetNodegroupKubernetesVersion(m.clientSet.CoreV1().Nodes(), s.Name)
-			if err != nil {
-				return nil, errors.Wrap(err, "getting nodegroup's kubernetes version")
-			}
+	// Create an empty array here so that an object is returned rather than null
+	summaries := []*Summary{}
+	for _, s := range stacks {
+		summary, err := m.stackToSummary(s)
+		if err != nil {
+			return nil, err
+		}
+		if summary != nil {
```

Comment:
Don't need the nil check. Append doesn't do anything if it's nil.

Reply:
i did not know that! wow

Reply:
wait thats not true! https://go.dev/play/p/O4nfw0O-yn7

Reply:
Oh! So sorry, I misread this part. :( I thought it's a slice being added. Bah.